src/net/ossl.cc

```diff
@@ -814,17 +815,17 @@ class session : public enable_shared_from_this<session>, public session_impl {
                 _eof = true;
                 return make_ready_future<stop_iteration>(stop_iteration::yes);
             }
-            auto err = make_exception_ptr(ossl_error::make_ossl_error(
-                "Error occurred during SSL write"));
+            auto err = ossl_error::make_ossl_error(
+                "Error occurred during SSL write");
             return handle_output_error(err).then([] {
```

Comment:
question: should these be `std::move(err)` something?

Reply:
Probably not a bad idea to save some space